src/librbd/cache/rwl/InitRequest.h

```diff
@@ -50,6 +56,7 @@ class InitRequest {
   InitRequest(ImageCtxT &image_ctx, Context *on_finish);
 
   ImageCtxT &m_image_ctx;
+  cache::ImageCache<ImageCtx> *m_image_cache;
```

Comment:
Nit: now that IO is dispatch via the `ImageDispatchInterface`, the RWL/PWL cache no longer needs to inherit from `ImageCache`

Reply:
Removed the inheritance from ImageCache.